packages/nimble-components/src/table/tests/table.spec.ts

```diff
@@ -215,6 +215,20 @@ describe('Table', () => {
             expect(pageObject.getHeaderTitle(0)).toBe(headerContents);
         });
 
+        it('sets title when header text is ellipsized in a span', async () => {
+            const headerContents = 'a very long value that should get ellipsized due to not fitting within the default header width';
+            await element.setData(simpleTableData);
+            await connect();
+            await waitForUpdatesAsync();
+            const span = document.createElement('span');
+            span.textContent = headerContents;
+            element.columns[0]!.textContent = '';
+            element.columns[0]!.appendChild(span);
```

Comment:
wanna show off to your friends? (almost definitely ignore)

```suggestion
            element.columns[0]!.replaceChildren(Object.assign(document.createElement('span'), {textContent: headerContents}))
```

Reply:
Ignoring :P